pkg/eestream/rs_test.go

```diff
@@ -156,7 +156,7 @@ func TestRSEncoderInputParams(t *testing.T) {
 		data := randData(32 * 1024)
 		fc, err := infectious.NewFEC(2, 4)
 		if !assert.NoError(t, err, errTag) {
```

Comment:
You can use `assert.Error(t, err, errTag)`

Reply:
No. `!assert.NoError(...)` is different than `assert.Error(...)`.

I want to assert that there is no error, but if this check fails, the execution for this test case must stop (mostly to avoid panic and other wierd things). `assert.Error(...)` would pass only if there is an error, which is not what I want here.